pyrenew_hew/utils.py

```diff
@@ -1,9 +1,54 @@
-import numpy as np
+import jax.numpy as jnp
 
 
 def convert_to_logmean_log_sd(mean, sd):
-    logmean = np.log(
-        np.power(mean, 2) / np.sqrt(np.power(sd, 2) + np.power(mean, 2))
+    logmean = jnp.log(
+        jnp.power(mean, 2) / jnp.sqrt(jnp.power(sd, 2) + jnp.power(mean, 2))
     )
-    logsd = np.sqrt(np.log(1 + (np.power(sd, 2) / np.power(mean, 2))))
+    logsd = jnp.sqrt(jnp.log(1 + (jnp.power(sd, 2) / jnp.power(mean, 2))))
     return logmean, logsd
+
+
+def normed_shedding_cdf(
```

Comment:
more descriptive names (or more comments) would be very helpful for understanding this code.